examples/mbedTLS/CMakeLists.txt

```diff
@@ -0,0 +1,72 @@
+cmake_minimum_required(VERSION 3.0)
+
+# Emulate HunterGate:
+# * https://github.com/hunter-packages/gate
+include("../common.cmake")
+
+project(download-mbedtls)
+
+hunter_add_package(mbedtls)
+find_library(
```

Comment:
> Looks like there is already an issue for the same, ARMmbed/mbedtls#298

Yes, seems the same

> Please use mbedtls-2.5.2 branch

Done, branch `hunter-2.5.2`.

> Alright, but that might take some time

There are a lot of boilerplate code but it's not very hard actually.
Take a look at these examples:
* https://github.com/forexample/package-example/blob/master/Foo/CMakeLists.txt
* https://github.com/cgold-examples/fruits